tests/Nox.Generator.Tests/Application/ApplicationEventTests.cs

```diff
@@ -7,15 +8,15 @@
 
 namespace Nox.Generator.Tests.Application;
 
-public class ApplicationEventTests: IClassFixture<GeneratorFixture>
+public class ApplicationEventTests : IClassFixture<GeneratorFixture>
```

Comment:
```suggestion
public class IntegrationEventTests : IClassFixture<GeneratorFixture>
```

Reply:
Changes were applied. Thanks for the suggestion.